### ISSUE TITLE
Accuracy at first few Step

### ISSUE THREAD

**Issue by ducminhnguyen** [hx173149/C3D-tensorflow#36]
Hi,

The accuracy at the first 300 step is 0.1000 or 0.0500. Is this normal or is there something wrong?

Thank you

**Comment by Tream733** [comment]
have you solved the problem？i got a result just like your‘s.it's normal?

**Comment by ducminhnguyen** [comment]
@Tream733 nope, did you leave it to train for a few more iter. Loading the pretrained model seems to work. But I dont understand the logic. It should improve gradually.

**Comment by Tream733** [comment]
@ducminhnguyen i have finished all the 5000 step.but it's result not very good.i will send  the tensorboard picture to your email。i will try to use  some different parameter in my follow work 。will you continue do this work？if you continue，can i talk more to you about this work in the following time？

**Comment by ducminhnguyen** [comment]
@Tream733 Yeah, I will continue working on C3D for a while. Thanks in advance for the tensorboard pics. I   think the init function of the training process have problem. With the right param (pretrained) this works very well, but if we train from scratch it does not work?

**Comment by geroge-gao** [comment]
@ducminhnguyen have you ever solved the problem?I met the same problem,and do you know how to use the pre-trained model?

**Comment by ducminhnguyen** [comment]
yeah, I left t over night and the result come out good the next day.

**Comment by geroge-gao** [comment]
@ducminhnguyen did you train the model from the scratch or by using the pre-trained model provided by hx173149?

**Comment by ducminhnguyen** [comment]
I trained from scratch

**Comment by FENGShuanglang** [comment]
@geroge-gao your result is 1.000,Why do you think he is not very good，

**Comment by geroge-gao** [comment]
@FENGShuanglang because the validation accuracy is only 0.023231257

**Comment by geroge-gao** [comment]
@FENGShuanglang  I have changed it, the model in predict_c3d_ucf101.py is what I trained myself. but it doesn't work.

**Comment by FENGShuanglang** [comment]
@geroge-gao mybe you should try reducing the num of training iterations

**Comment by geroge-gao** [comment]
@FENGShuanglang Actually I train the C3D from scratch，but the accuracy is always 1.0000 .you can see #59

**Comment by FENGShuanglang** [comment]
you should trian it use pre-trained model ,it is very hard and no-rubustness if you train the C3D from scratch .why do you train it from scratch?by the way ,i have to go to chifan! see you later

**Comment by geroge-gao** [comment]
@FENGShuanglang so how do you use the pre-trained model? as I know ,the tensorflow model contains four files:  .meta  , .index , .data and the checkpoint files. The author only provided the .model file. i guess it's Correspondied to the .data file. so I use the .model file to cover the .data file simply. But it still doesn't work.

**Comment by geroge-gao** [comment]
@FENGShuanglang so how do you use the pre-trained model? as I know ,the tensorflow model contains four files: .meta  .index  .data and the checkpoint files. The author only provided the .model file. i guess it corresponds to the .data file. so I use the .model file to cover the .data file simply. But it still doesn't work.

**Comment by geroge-gao** [comment]
I did as what you said,but there occurred an error
`
ValueError:` The specified path: conv3d_deepnetA_sport1m_iter_1900000_TF.model is a file. Please specify only the path prefix to the checkpoint files.
`
you can see #10 ,
as for "use the .model file to cover the .data file simply", actually I first train the C3D from the scratch, when there will be four files in the models foder. then I replace the .data-00000-of-00001 with .model file.

**Comment by geroge-gao** [comment]
I did as what you said,but there occurred an error

`ValueError: The specified path: conv3d_deepnetA_sport1m_iter_1900000_TF.model is a file. Please specify only the path prefix to the checkpoint files.`
you can see #10 ,
as for "use the .model file to cover the .data file simply", actually I first train the C3D from the scratch, when there will be four files in the models foder. then I replace the .data-00000-of-00001 with .model file.

**Comment by FENGShuanglang** [comment]
You can try to change a pre-training network，for example “sports1m_finetuning_ucf101.model”

**Comment by geroge-gao** [comment]
I have tried all the three .model files ,but it's still the same, no any change.

**Comment by FENGShuanglang** [comment]
I don't know why ,but you can reference https://zhuanlan.zhihu.com/p/35388036 and Ask the blogger about your question,he is my classmate

**Comment by geroge-gao** [comment]
It seems a normal issue on windows, if you train it on Linux, it won't happen. hope it will help you.
